azavea/modellab-geoprocessing
# Include Static Configuration in .jar File

The default layers and breaks are now registered automatically.

Connects #6

**To Test**
   * Type `./sbt assebmly` while in this branch to create a jar file
   * Transfer the new job file to the worker `scp ./target/scala-2.10/modellab-geoprocessing-assembly-0.4.0.jar vagrant@33.33.42.82:`
   * `ssh` to the worker and copy the jar file to `/opt/modellab-geoprocessing/`
   * Pull down this modellab branch: https://github.com/jamesmcclain/modellab/tree/feature/no-static-config and reprovision the worker on the branch
   * Optionally reload the worker
   * Try the site, it should behave as before
   * Also confirm that you can post layers and breaks to the service; that that capability still works
